mesh/ncmesh.hpp

```diff
@@ -145,7 +148,10 @@ class NCMesh
        Note that if anisotropic refinements are present in the mesh, some of the
        derefinements may have to be skipped to preserve mesh consistency. */
    virtual void Derefine(const Array<int> &derefs);
+   virtual void GetFineToCoarsePartitioning(const Array<int> &derefs,
+                                            Array<int> &new_ranks) {}
 
+   int GetNGhostElements() const { return 0; }
```

Comment:
Is this needed in the serial class? Seems that you only use the parallel class version.